### DISCUSSION TITLE
MTB in english but with "french" date / time format

### DISCUSSION THREAD

**Discussion by mff73** [mytourbook/mytourbook#1299] [Q&A]
Is it possible ?
I like using MTB with menus in english (to share here for example), but dates and durations are in english format as well, which i really don't like.
Thanks

**Comment by wolfgang-ch** [comment]
I'm using english with the britisch date/time format

`"C:\Program Files\MyTourbook\mytourbook.exe" -nl en_GB`

**Comment by mff73** [comment]
Thanks, but i would like to use english for menus, BUT non british date/time.
i tried -nl en_FR, it doesn't work.
i tried -nl fr, everything is in french.
is it not possible ?

**Reply by wolfgang-ch** [reply]
> i tried -nl en_FR, it doesn't work.

`en_FR` is not an international locale and it would be a lot of work to make the date/time locale customizable

https://stackoverflow.com/questions/8971593/setlocale-with-value-en-fr#answer-8972570

![image](https://github.com/mytourbook/mytourbook/assets/1283445/90abde79-e615-49b1-8b45-833b9e51e41f)

**Reply by mff73** [reply]
Ok, no problem, i will choose en or fr.
thanks.
